RELEASE_NOTES.md

```diff
@@ -7,8 +7,8 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# azure-chef-extension 1210.12.4.1000 release notes:
-This release of azure-chef-extension fixes the issue of extension 1210.12.4.1 not available in all locations for ARM commands. Extension 1210.12.4.1 has been deleted.
+# azure-chef-extension 1210.12.5.1000 release notes:
+This release of azure-chef-extension decouples extension version from the chef-client version. The extension now installs the latest version of chef-client.
 

```

Comment:
How about mentioning here that this version won't work in VNET where internet access not available? As apt-repo need public connectivity ?